index.js

```diff
@@ -148,88 +111,93 @@ bot.on("guildDelete", (guild) => {
 });
 
 /**
- * Cron Job: 08:00:00 AM Europe/Zagreb - Go through each member and ask for standup
+ * This function loops through the days pre-configured on the `__config__.js` file and sets up the stand ups on those .
  */
-schedule.scheduleJob(
-  { hour: 8, minute: 0, dayOfWeek: new schedule.Range(1, 5), tz: "Europe/Zagreb" },
-  (time) => {
-    console.log(`[${time}] - CRON JOB 1 START`);
-    promptMembers();
+for (day in config.days) {
```

Comment:
I think it's better to use cron expressions: https://github.com/NodeFactoryIo/standup-bot/pull/3

for config, you should be able to use node-schedule to print next trigger date for config file